h2o-r/h2o-package/R/models.R

```diff
@@ -3115,20 +3115,83 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+#'
+#' The H2ONode class empty definition to make setClassUnion possible, as H2ONode has circular dependency.
+#'
+setClass("H2ONode") # Avoid warning about undefined class, H2ONode has circular dependency
+setClassUnion("H2ONodeOrNULL",members=c("H2ONode", "NULL"))
```

Comment:
I wonder if instead of NULL there should be NA. I looked into our code and we use NULL.